example/csv/src/test/resources/sales/test.csv

```diff
@@ -0,0 +1,3 @@
+id,name,sex,age
```

Comment:
According to other CSV files, there should be a data type here. If it is the default String type, it is normal for Calcite to throw an exception.